TheWarWithin/RogueAssassination.lua

```diff
@@ -1126,19 +1032,19 @@ spec:RegisterAuras( {
     },
     -- Talent: Damage done increased by $w1%.
     -- https://wowhead.com/beta/spell=193641
-    elaborate_planning = {
+   --[[ elaborate_planning = {
```

Comment:
If this is gone, just delete it.

Reply:
I wasn't 100% sure at the time, but yeah it is gone.